jobbergate-agent/charmcraft.yaml

```diff
@@ -0,0 +1,39 @@
+name: jobbergate-agent
+title: The Jobbergate Agent life-cycle manager.
+type: charm
+summary: Jobbergate Agent package life-cycle encapsulated in a charm
+description: |
+  This charm provides the Jobbergate Agent component of Vantage installed by Snap.
+  It embeds the snap package in a charm app and wraps its life-cycle.
```

Comment:
```suggestion
  It embeds the snap package in a charm app and wraps its lifecycle.
```

Reply:
See my comment above. I'll change it to "life cycle".